examples/src/index.js

```diff
@@ -16,6 +16,26 @@ const AsyncDemo = () => {
     }
   };
 
+  const getResponse = async () => {
+    try {
+      const res = await captchaRef.current.getResponse();
```

Comment:
There is no need in using `await` since `getResponse()` returns synchronously.  Same for `getRespKey`.